README.md

```diff
@@ -18,6 +18,23 @@ npm install ember-statechart-component
 yarn add ember-statechart-component
 ```
 
+### Optional
```

Comment:
Why move this under optional?

It'd be reallly awkward to use this library without the rfc 756 behavior

Reply:
Fair. I understood that it's only needed for `state.matches` calls. Which is immensely useful. Just not _necessary_. But yeh, good defaults.